google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -2205,6 +2205,26 @@ public void testFastQueryHTTPException() throws InterruptedException {
     }
   }
 
+  @Test
+  public void testDmlStatistics() throws InterruptedException {
+    String tableName = TABLE_ID_FASTQUERY.getTable();
```

Comment:
Only concern here is whether this UPDATE will impact other tests that use this table, as test execution order is not guaranteed.

Reply:
I think this should be okay since the WHERE is set to TRUE so it will always just update the table (2 rows).